Formula/u/util-linux.rb

```diff
@@ -107,8 +107,10 @@ def install
     end
 
     system "./configure", *args, *std_configure_args
-    ENV.append "LDFLAGS", "-lm" if OS.linux?
-    system "make", "install"
+
+    install_args = []
+    install_args.append "LDFLAGS=\"-lm\"" if OS.linux?
```

Comment:
```suggestion
    install_args << "LDFLAGS=-lm" if OS.linux?
```
You don't need the extra quotes, and `<<` matches what other formulae do.